src/js/components/MesosLogView.js

```diff
@@ -197,6 +204,22 @@ class MesosLogView extends mixin(StoreMixin) {
     }
   }
 
+  goToNewHighlightedSearch() {
+    let logContainer = this.getLogContainerNode();
+    let node = logContainer.querySelector('.highlight.selected');
+    if (!node) {
+      return;
+    }
+
+    let containerHeight = logContainer.clientHeight;
+    let containerScrollTop = logContainer.scrollTop;
+
+    if ((node.offsetTop > containerHeight + containerScrollTop) ||
+      node.offsetTop < containerScrollTop) {
```

Comment:
I can't remember, but are both `offsetTop` and `scrollTop` well-supported in all browsers?